models/demos/t3000/falcon40b/tt/falcon_model.py

```diff
@@ -308,7 +308,7 @@ def fwd_prefill(
             layer_output = layer_output[0]
 
         if layer_output.dtype != self.model_config["BFP8_DTYPE"]:
-            layer_output = ttnn.experimental.typecast(
+            layer_output = ttnn.typecast(
```

Comment:
this can to stay ttnn.experimental.typecast. it refers to typecast in ttnn/operations/experimental/copy